client/src/pages/ProfileListings/ProfileCard.tsx

```diff
@@ -0,0 +1,56 @@
+import RoomIcon from '@material-ui/icons/Room';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import Avatar from '@material-ui/core/Avatar';
+import Rating from '@material-ui/lab/Rating';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import useStyles from './useStyles';
+
+import { User } from './dummyUserData';
+
+interface ProfileCardProps {
+  user: User;
+}
+
+const ProfileCard: React.FC<ProfileCardProps> = ({ user }) => {
+  const classes = useStyles();
+
+  const handleClickCard = () => {
+    // handle navigating to profile page here
+  };
+
+  return (
+    <Card className={classes.card} onClick={handleClickCard} raised={true}>
+      <CardActionArea>
+        <CardContent className={classes.cardContentUpper}>
+          <Avatar className={classes.cardAvatar} src={user.image} alt="" />
+          <Typography gutterBottom variant="h5" component="h2" style={{ fontWeight: 'bold', marginTop: '.5rem' }}>
```

Comment:
for the sake of consistency, let's stick with using the same styling pattern with `makeStyles` as in the other files